lib/chains/networkRequestChain.js

```diff
@@ -96,21 +89,21 @@ const toJSON = data => {
 
 	// Apply URL comparator
 	condition.type = applyNegation(condition.type, data);
-	subject.compare = SUBJ_COMPARATOR_M[data.comparator.type];
+	subject.compare = data.comparator.type;
 	subject.val = data.comparator.val;
 
 	// Apply request parameters
 	subject.requestInfo = (data.request && data.request.props || []).map(item => ({
-		name: typeof item.name === 'symbol' ? NETWORK_PROP_M[item.name] : item.name,
-		val: item.name === NETWORK_PROP.METHOD ? NETWORK_METHOD[item.val] : item.val,
-		compare: PROP_COMPARATOR_M[item.compare],
+		name: item.name,
```

Comment:
Do we actually need to map in this case? Are there any other properties in item?